spec/openapi.yaml

```diff
@@ -81,7 +79,7 @@ tags:
   - name: Services
     description: Operations about services
   - name: User
-    description: Operations about user
+    description: Operations about users
 security:
```

Comment:
It would be good to have some more verbose descriptions under as there are in the above sections, e.g Files:

- Search
- Schemas
- JSON Schemas
- Services
- User
- Health Check 
- Workflows